recipes/orjson/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex
+
+./install.sh --prefix=$PREFIX
```

Comment:
@EntPyle This line. Line 22 is already doing what we want.

Reply:
Got it.